meta/lib/oeqa/selftest/cases/devtool.py

```diff
@@ -2212,7 +2212,7 @@ def __devtool_ide_recipe(self, recipe_name, build_file, testimage):
         conf_lines = [
             'IMAGE_CLASSES += "image-combined-dbg"',
             'IMAGE_GEN_DEBUGFS = "1"',
-            'IMAGE_INSTALL += "gdbserver %s-ptest"' % recipe_name
+            'PACKAGE_INSTALL:append = " gdbserver %s-ptest"\n' % recipe_name
```

Comment:
not sure what't the issue here:

- Using the += operator as mentioned in the commit messages was fixed long time ago. The commit message looks outdated.
- Using PACKAGE_INSTALL instead of IMAGE_INSTALL add gdbserver also to special images like initramfs or the test-empty-image. This is probably not good.

I guess this commit is not needed.

Reply:
I looks like you are running a pretty old version of my patches. This has been fixed several weeks ago.

- [v6](https://patchwork.yoctoproject.org/project/oe-core/patch/20230910161850.4032227-12-adrian.freihofer@siemens.com/) is with append. It's from one month ago.
- [v5](https://patchwork.yoctoproject.org/project/oe-core/patch/20230905165913.2560907-14-adrian.freihofer@siemens.com/) has the bug

Reply:
Just to be sue, here is the latest from today Oct 9 2023: https://git.yoctoproject.org/poky-contrib/log/?h=adrianf/devtool-ide